### PR TITLE
Python content: add Exceptions

### PR DESCRIPTION
**Pull request by rollandf**

### CODE REVIEW

File: docs/course_materials/exceptions.html [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+          </code></pre>
+        </section>
+        <section>
+          <h3><b>Accessing the Exception</b></h3>
+           <ul>
+            <li>Use "as" to access the exception object.</li>
+          </ul>
+<pre><code data-trim data-noescape class="hljs">
+try:
+    raise RuntimeError("o hai")
```
Comment (sleviim) [inline docs/course_materials/exceptions.html]: what?

File: docs/course_materials/exceptions.html [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+    print('Outer')
+          </code></pre>
+            <ul>
+            <li>The (default) root exception handler terminates the Python process.</li>
+          </ul>
+        </section>
+        <section>
+          <h3><b>Propagating Exceptions</b></h3>
+          <ul>
+            <li>Propagation can be forced by using raise without arguments</li>
```
Comment (sleviim) [inline docs/course_materials/exceptions.html]: maybe 'raise' , or bolding

File: docs/course_materials/exceptions.html (hunk 1 above)
Comment (rollandf) [reply docs/course_materials/exceptions.html]: Changed to "oops"

File: docs/course_materials/exceptions.html (hunk 2 above)
Comment (rollandf) [reply docs/course_materials/exceptions.html]: changed to 'raise'

**Pull request merged**